docs/complete_configuration_guide.md

```diff
@@ -0,0 +1,353 @@
+Zat detailed configuration guide
+---
+
+Zat automatically copies your Zoom recordings to the Google Drive folder of your choices - you no longer need to download and copy file over, Zat will do it for you!
+In this detailed guide, you'll get to install and configure Zat - it seems like a lot of steps, but you'll only need to do it once!
+
+## Pre-requisites
+
+We'll be using the Terminal app to run commands.
+You can already launch it by hitting the `command+space` keys then searching for `terminal`.
+
+### Install Docker and configure it
+
+1. Download and install [Docker Desktop](https://docs.docker.com/desktop/install/mac-install/) for Mac.
+You'll get a warning saying that Docker Desktop needs privileged access - you will need to enter your computer password when prompted.
+You'll see a "Docker Subscription Service Agreement" screen - accept it.
+
+2. Start Docker and create an account
+Open the Docker app, and follow the steps to create an account.
+
+
+### Log into Elastic container registry
+
+We're now going to log into Elastic container registry, where we'll download zat.
+Visit [https://docker-auth.elastic.co](https://docker-auth.elastic.co) - click on the `Login to @elastic with GitHub` button.
+You may get redirect to an Okta log-in screen, but eventually will see a page as below:
+
+![Container Registry Login](img/container_registry_loging.png)
+
+
+Copy the full line, after the `$` sign and paste it into the Terminal window and press enter.
+You should see a `Login Succeeded` message.
+
```

Comment:
```suggestion
```

This is a high barrier too, why not just docker hub since this is all public anyway?